docs/src/pages/docs/crocks/Identity.md

```diff
@@ -333,12 +333,7 @@ import Identity from 'crocks/Identity'
 
 Identity(42)
   .valueOf()
-//=> 33
-
-Identity(20)
-  .concat(Identity(22))
-  .valueOf()
```

Comment:
I can see that this won't work, lets add in an example that does work. Each `Identity` will need to contain a SemiGroup of the same type

Reply:
Yeah both of those work great, i'll leave it to you to pick your fav, only request would be to copy similiar formatting to the one that didn't work

Reply:
Sure I will Thanks

Reply:
@dalefrancis88 Please approve my PR #472